### ISSUE TITLE
Please make multi-select + DEL-key work in bones list of Weights panel

### ISSUE THREAD

**Issue by bencresty** [EsotericSoftware/spine-editor#827]
When wanting to remove all bones from the bones list in the weight panel this is what we need to do now:
- Select a bone
- Move your pen all the way to the bottom to press 'Remove' (Because the DEL-key on the keyboard doesn't work on this, it's removing something else, like an attachment instead when we press that)
- Select the next bone
- Move your pen all the way to the bottom to press 'Remove'
- etc. for all following bones

It would be a lot easier if we could just multi-select the bones in the list and hit 'remove'.
AND it would definitely be helpful too if we could just hit the DEL-key on the keyboard to remove the selected bone from the list. 

[edit] A 'clear list'-button would also be helpful.

Spine 4.2.3 pro

**Comment by misaki-eymard** [comment]
@bencresty You can specify a hotkey for the `Remove` button in the hotkey.txt file::
![weights-remove-hotkey](https://github.com/user-attachments/assets/b23fc907-78ca-4e5b-a7a3-fae9acfc0d79)

This allows you to assign any hotkey, so I'll close this ticket.

**Comment by bencresty** [comment]
> @bencresty You can specify a hotkey for the `Remove` button in the hotkey.txt file::

Yeah, but I can't use DEL for that, because than it would have a conflict with the 'viewport's DEL, where it tries to remove an attachment.  Finding another key which is completely different compared to other software is obviously not wanted and shouldn't be necesary IMO.

Wouldn't it make more sense just to auto detect that we're on the bones list (active windows element) and just make the DEL key work there, like other software does?

**Comment by NathanSweet** [comment]
In 4.3 you can multi select bones in the weights view. Clicking the remove button once is easy enough. We could support pressing delete when the weights view has focus, the only issue is if that might delete in the weights view when the user wanted to delete elsewhere.
